website/source/api/secret/identity/entity-alias.html.md

```diff
@@ -0,0 +1,219 @@
+---
+layout: "api"
+page_title: "Identity Secret Backend - HTTP API"
```

Comment:
Since we have multiple pages under identity, name each one differently:`Identity Secret Backend: Alias - HTTP API`